db/schema.sql

```diff
@@ -25,14 +25,24 @@ CREATE TABLE tasks(
     min_batch_duration     BIGINT NOT NULL,           -- the minimum duration in seconds of a single batch interval
     tolerable_clock_skew   BIGINT NOT NULL,           -- the maximum acceptable clock skew to allow between client and aggregator, in seconds
     collector_hpke_config  BYTEA NOT NULL,            -- the HPKE config of the collector (encoded HpkeConfig message)
-    agg_auth_key           BYTEA NOT NULL,            -- HMAC key used by this aggregator to authenticate messages to/from the other aggregator (encrypted)
-    hpke_config            BYTEA NOT NULL,            -- the HPKE config of this aggregator (encoded HpkeConfig message)
-    hpke_private_key       BYTEA NOT NULL             -- private key corresponding to hpke_config (hpke::HpkePrivateKey, encrypted)
+    agg_auth_key           BYTEA NOT NULL             -- HMAC key used by this aggregator to authenticate messages to/from the other aggregator (encrypted)
 
-    -- TODO(timg): move vdaf_verify_param, agg_auth_key, hpke_config, hpke_private_key to new
+    -- TODO(timg): move vdaf_verify_param, agg_auth_key to new
     -- tables with many:1 relationships to tasks to allow for rotation of secrets
 );
 
+-- The HPKE public keys (aka configs) and private keys used by a given task.
+CREATE TABLE task_hpke_keys(
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,  -- artificial ID, internal-only
+    task_id BIGINT NOT NULL, -- XXX: document
```

Comment:
(one day I will learn to search the entire repo, and not just the `src/` directory, when looking for stray `XXX` markers...)

Reply:
Here's a oneliner for a pre-commit hook: `bash -c '! git grep XXX'`. (The bash subprocess is presumably necessary because `set -e` only cares about subprocesses, and not expression evaluation)